src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -37,6 +37,7 @@
                                          sed_a_factor
                                          )
 
+import time
```

Comment:
Probably worth moving this import toward the top of the file, above the imports from the application.